<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbSettingsDialog</class>
 <widget class="QDialog" name="dbSettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Database Connection Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>350</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUserName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHostName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>5432</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUserName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>user name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHostname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>host name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>port</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDatabase">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>database name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetPort">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>180</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseDatabases">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>280</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblConnectionName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>connection name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtConnectionName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestConnection">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>230</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Connection</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxDBs">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtConnectionName</tabstop>
  <tabstop>txtUserName</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>txtHostName</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>btnResetPort</tabstop>
  <tabstop>btnTestConnection</tabstop>
  <tabstop>cbxDBs</tabstop>
  <tabstop>btnBrowseDatabases</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dbSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dbSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
